client/src/views/AssetsList.tsx

```diff
@@ -51,7 +51,7 @@ type Props = {
 function AssetsList(props: Props): JSX.Element {
   const { classes } = useStyles();
   const { assets, headerContentRight, headerText } = props;
-  if (!assets) return <> </>;
+  if (!assets || !Array.isArray(assets)) return <> No Results</>;
```

Comment:
assets was coming back as an error object when DB is unseeded, so I added a check for if it is an array.